### PR TITLE
Assignment2 Completed

### PR DESCRIPTION
**Pull request by tinotrifan**

### CODE REVIEW

**Comment by ocorcho** [comment]
Exacto, no deberían ser recursos sino literales

File: Assignment2/tinotrifan.rdf [hunk 1]
```diff
@@ -1,24 +1,24 @@
...
+    </rdf:Description>
+	<rdf:Description rdf:about="#Measurement8401">
+	<hasTemperature rdf:resource="#29"/>
+	<atTime rdf:datatype="http://www.w3.org/2001/XMLSchema#date">2010-06-12T12:00:12</atTime>
+    </rdf:Description>
+    <rdf:Description rdf:about="#Computer101">
+	<hasOwner rdf:resource="#User10A"/>
+    </rdf:Description>
+    <rdf:Description rdf:about="#User10A">
+	<hasName rdf:resource="#Pedro"/>
```
Comment (ocorcho) [inline Assignment2/tinotrifan.rdf]: This should be a string, not a resource.

File: Assignment2/tinotrifan.rdf [hunk 2]
```diff
@@ -1,24 +1,24 @@
...
+    xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#">
+    <rdf:Description rdf:about="#Class01">
+	<includes rdf:resource="#Sensor029"/>
+	<includes rdf:resource="#Computer101"/>
+    </rdf:Description>
+    <rdf:Description rdf:about="#Sensor029">
+	<hasMeasurement rdf:resource="#Measurement8401"/>
+    </rdf:Description>
+	<rdf:Description rdf:about="#Measurement8401">
+	<hasTemperature rdf:resource="#29"/>
```
Comment (ocorcho) [inline Assignment2/tinotrifan.rdf]: This should be a string, not a resource.

File: Assignment2/tinotrifan.ttl [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+
+<http://www.ontogrid.net/StickyNote#Class01> ns0:includes ns0:Sensor029, ns0:Computer101 .
+ns0:Sensor029 ns0:hasMeasurement ns0:Measurement8401 .
+ns0:Measurement8401
+  ns0:hasTemperature ns0:29 ;
+  ns0:atTime "2010-06-12T12:00:12"^^xsd:date .
+
+ns0:Computer101 ns0:hasOwner ns0:User10A .
+ns0:User10A ns0:hasName ns0:Pedro .
```
Comment (ocorcho) [inline Assignment2/tinotrifan.ttl]: Pedro should be a string

File: Assignment2/tinotrifan.ttl [hunk 4]
```diff
@@ -0,0 +1,12 @@
+
+@prefix ns0: <http://www.ontogrid.net/StickyNote#> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+
+<http://www.ontogrid.net/StickyNote#Class01> ns0:includes ns0:Sensor029, ns0:Computer101 .
+ns0:Sensor029 ns0:hasMeasurement ns0:Measurement8401 .
+ns0:Measurement8401
+  ns0:hasTemperature ns0:29 ;
+  ns0:atTime "2010-06-12T12:00:12"^^xsd:date .
```
Comment (ocorcho) [inline Assignment2/tinotrifan.ttl]: Should be dateTime

File: Assignment2/tinotrifan.ttl [hunk 5]
```diff
@@ -0,0 +1,12 @@
+
+@prefix ns0: <http://www.ontogrid.net/StickyNote#> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+
+<http://www.ontogrid.net/StickyNote#Class01> ns0:includes ns0:Sensor029, ns0:Computer101 .
+ns0:Sensor029 ns0:hasMeasurement ns0:Measurement8401 .
+ns0:Measurement8401
+  ns0:hasTemperature ns0:29 ;
```
Comment (ocorcho) [inline Assignment2/tinotrifan.ttl]: No prefix needed for 29

Comment (tinotrifan) [reply Assignment2/tinotrifan.ttl]: fixed

File: Assignment2/tinotrifan.ttl (hunk 4 above)
Comment (tinotrifan) [reply Assignment2/tinotrifan.ttl]: fixed

File: Assignment2/tinotrifan.ttl (hunk 3 above)
Comment (tinotrifan) [reply Assignment2/tinotrifan.ttl]: ficed

Comment (tinotrifan) [reply Assignment2/tinotrifan.ttl]: fixed

File: Assignment2/tinotrifan.rdf (hunk 1 above)
Comment (tinotrifan) [reply Assignment2/tinotrifan.rdf]: fixed

File: Assignment2/tinotrifan.rdf (hunk 2 above)
Comment (tinotrifan) [reply Assignment2/tinotrifan.rdf]: fixed

File: Assignment2/gemawari-GemaGonzalez.rdf [hunk 6]
```diff
@@ -1,25 +1,25 @@
...
+	<hasTemperature rdf:resource="#29"/>
+	<atTime rdf:datatype="http://www.w3.org/2001/XMLSchema#date">2010-06-12T12:00:12</atTime>
+    </rdf:Description>
+    <rdf:Description rdf:about="#Computer101">
+	<hasOwner rdf:resource="#User10A"/>
+    </rdf:Description>
+    <rdf:Description rdf:about="#User10A">
+	<hasName rdf:resource="#Pedro"/>
+    </rdf:Description>
+    <rdf:Description rdf:about="#Pedro">
```
Comment (ocorcho) [inline Assignment2/gemawari-GemaGonzalez.rdf]: Check other student's submission. This is wrong as this should be a literal

Comment (ocorcho) [reply Assignment2/gemawari-GemaGonzalez.rdf]: Ahh, I see that this is from anotehr student.

File: Assignment2/tinotrifan.ttl [hunk 7]
```diff
@@ -0,0 +1,11 @@
...
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+
+<http://www.ontogrid.net/StickyNote#Class01> ns0:includes ns0:Sensor029, ns0:Computer101 .
+ns0:Sensor029 ns0:hasMeasurement ns0:Measurement8401 .
+ns0:Measurement8401
+  ns0:hasTemperature 29 ;
+  ns0:atTime "2010-06-12T12:00:12"^^xsd:dateTime .
+
+ns0:Computer101 ns0:hasOwner ns0:User10A .
+ns0:User10A ns0:hasName "\"Pedro\"" .
```
Comment (ocorcho) [inline Assignment2/tinotrifan.ttl]: You should remove the escaping of double quotes.

File: Assignment2/tinotrifan.rdf [hunk 8]
```diff
@@ -0,0 +1,23 @@
...
+    </rdf:Description>
+	<rdf:Description rdf:about="#Measurement8401">
+	<hasTemperature rdf:datatype="http://www.w3.org/2001/XMLSchema#integer">29</hasTemperature>
+	<atTime rdf:datatype="http://www.w3.org/2001/XMLSchema#dateTime">2010-06-12T12:00:12</atTime>
+    </rdf:Description>
+    <rdf:Description rdf:about="#Computer101">
+	<hasOwner rdf:resource="#User10A"/>
+    </rdf:Description>
+    <rdf:Description rdf:about="#User10A">
+	<hasName>"Pedro"</hasName>
```
Comment (ocorcho) [inline Assignment2/tinotrifan.rdf]: Remove the double quotes.

File: Assignment2/gemawari-GemaGonzalez.rdf (hunk 6 above)
Comment (tinotrifan) [reply Assignment2/gemawari-GemaGonzalez.rdf]: fixed

File: Assignment2/tinotrifan.rdf (hunk 8 above)
Comment (tinotrifan) [reply Assignment2/tinotrifan.rdf]: fixed

File: Assignment2/tinotrifan.ttl (hunk 7 above)
Comment (tinotrifan) [reply Assignment2/tinotrifan.ttl]: fixed

**Pull request merged**